program/find-the-eigenvalues-of-a-matrix/find_the_eigenvalues_of_a_matrix_using_scala.scala

```diff
@@ -0,0 +1,16 @@
+import org.apache.commons.math3.linear.{EigenDecomposition, RealMatrix, MatrixUtils}
+
+object EigenvaluesCalculator {
+  def main(args: Array[String]): Unit = {
+    
+    val matrixData = Array(Array(1.0, 2.0), Array(3.0, 4.0))
+    val matrix: RealMatrix = MatrixUtils.createRealMatrix(matrixData)
+
```

Comment:
@Iqbalmde The ask is for a 3 x 3 matrix . 
Can you please test it for a 3 x 3 matrix.